debezium-connector-mongodb/src/main/resources/io/debezium/connector/mongodb/sink/build.version

```diff
@@ -0,0 +1 @@
+version=${project.version}
```

Comment:
Do we need a separate build version for sink connector when the source and sink ar ein the same package?

Reply:
No. Agreed.